ci_framework/roles/devscripts/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Gathering the configuration
+  when: cifmw_devscripts_config is not defined
```

Comment:
Note that, if this is passed as a parameter, any call to `set_fact` in order to update it will fail

Reply:
Ack. Removed the check for the variable being defined. Also ensured, that the variable is not documented in Readme.md

The expectation is that any kind of override would come via `cifmw_devscripts_config_overrides`